source/features/add-branch-buttons.js

```diff
@@ -90,10 +91,10 @@ async function getDefaultBranchLink() {
 	);
 }
 
-export default async function () {
+async function init() {
 	const container = select('.file-navigation');
 	if (!container) {
-		return;
+		return false;
```

Comment:
What about moving this part to "dependencies" to keep them together:

```js
features.add({
	id: 'add-branch-buttons',
	dependencies: [
		features.isRepo.not(() => !select('.file-navigation'))
	],
	load: features.safeOnAjaxedPages,
	init
});
```

Reply:
That `select` is part of the feature, the returned value is used later. Using that just brings too much noise.